exercises/practice/hexadecimal/.meta/Example.roc

```diff
@@ -0,0 +1,23 @@
+module [parse]
+
+parseNibble = \char ->
+    when char is
+        c if c >= '0' && c <= '9' -> Ok (c - '0' |> Num.toU64)
+        c if c >= 'A' && c <= 'F' -> Ok (c - 'A' + 10 |> Num.toU64)
+        c if c >= 'a' && c <= 'f' -> Ok (c - 'a' + 10 |> Num.toU64)
+        _ -> Err InvalidNumStr
```

Comment:
Because this isn't doing any pattern matching I would suggest using an if else chain instead.

Reply:
Indeed, thanks!